src/tables/enemies.js

```diff
@@ -23,6 +23,14 @@
   }
 */
 
-const enemies = [[0, 1.5, 10, 'red']]
+const enemies = [
+  [0, 1.5, 10, 'red', 50],
+
+  /**
+   * enemy: wraith
+   * @see https://github.com/straker/js13k-2022/issues/3#issuecomment-1223405705
+   */
+  [1, 1, 10, 'white', 150]
```

Comment:
Enemies should also take from the `weapons` table to give them a weapon with damage and rate of fire. Then the weapon can define the projectile that is used. This would also let you define the effects for the projectile that get applied, such as the rooting effect on _n_ attacks. We'll need a way to differentiate between effects on attack and effects on hit (so probably a 2nd effects array).

Reply:
Just realized I already have this distinction. The weapon has an effects array and those will be for effects on attack. The projectile also has effects array and those should be for effects on hit.